big_tests/tests/sm_SUITE.erl

```diff
@@ -1,22 +1,38 @@
+%% Session Management tests
```

Comment:
Not needed
```suggestion
```

Reply:
omg, actually needed.
It is **Stream** Management.

There are two things called SM in MongooseIM (Stream and Session).